themes/qgrid/source/js/menu.js

```diff
@@ -1,3 +1,4 @@
+﻿var nav, li;
```

Comment:
this is bad design to have global variables like, even copy paste of selector is better